### ISSUE TITLE
Validation Accuracy is very low

### ISSUE THREAD

**Issue by bnaman50** [MadryLab/robust_representations#7]
Hey, 

Thanks for providing this robust model. I was using it for my experiments when I noticed that the top-1 accuracy on the validation set is around 3% (1555/50000 to be precise). But in the [paper](https://arxiv.org/pdf/1906.09453.pdf), it is mentioned that the accuracy is around 45%
![image](https://user-images.githubusercontent.com/5251592/62567611-268ac080-b851-11e9-851b-cde156f949ab.png)

It would be great if you could help me verify it. 

P.S. - I am using the ImageNet model provided in [this](https://github.com/MadryLab/robustness_applications?) repository. 

Thanks, 
Naman
